### PR TITLE
EzSettings Implementation

### PR DESCRIPTION
**Pull request by sveredyuk**
Guys, I found some time and start to making new open source -> [ez-engines](https://github.com/ez-engines)

It's my approach of builing Rails apps as a modular constructor system.

First of them -> [ez-settings](https://github.com/ez-engines/ez-settings) is super-easy tool to add trivial settings to your rails app. More information in ez-settings readme.

Just watch this video:
https://youtu.be/qApwWQIQQEc

### CODE REVIEW

**Comment by sveredyuk** [comment]
If none response in 72h I am mot responsible for myself (merge)

**Review by kushniryb — COMMENTED** [review]
Wow, this is rly good! Well done @sveredyuk :+1:

**Review by kushniryb — APPROVED** [review]

**Comment by sveredyuk** [comment]
@kushniryb 🥇

File: db/migrate/20162812090707_add_events_limit.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 class AddEventsLimit < ActiveRecord::Migration[5.0]
   def change
-    add_column :events, :limitation, :integer, default: Event::DEFAULT_LIMIT
+    add_column :events, :limitation, :integer, default: 50
```
Comment (floor114) [inline db/migrate/20162812090707_add_events_limit.rb:3]: There is not possible to get settings `Ez::Settings[:app, :events, :default_limit]` here?

File: spec/features/admin/events/create_spec.rb [hunk 2]
```diff
@@ -5,8 +5,8 @@
   end
 
   context 'dateTime inputs' do
-    let(:default_started_at_hour) { Event::DEFAULT_STARTED_AT_HOURS }
-    let(:default_finished_at_hour) { Event::DEFAULT_FINISHED_AT_HOURS }
+    let(:default_started_at_hour)  { Ez::Settings[:app, :events, :default_started_at_hours].to_i }
```
Comment (floor114) [inline spec/features/admin/events/create_spec.rb:8]: Why do we need `to_i` if `default_started_at_hours` type is integer?

**Comment by floor114** [comment]
As @kushniryb said, looks really good. Great job @sveredyuk.

**Review by floor114 — DISMISSED** [review]

**Review by floor114 — APPROVED** [review]

File: spec/features/admin/events/create_spec.rb (hunk 2 above)
Comment (sveredyuk) [reply spec/features/admin/events/create_spec.rb:8]: In 0.1 version I not implemented coercions. So all settings return as strings.

File: db/migrate/20162812090707_add_events_limit.rb (hunk 1 above)
Comment (sveredyuk) [reply db/migrate/20162812090707_add_events_limit.rb:3]: I don't want to make migrations depends on pluggable component

**Pull request merged**